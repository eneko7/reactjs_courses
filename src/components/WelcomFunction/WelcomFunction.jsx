import React from 'react';

const WelcomFunction = () => (
  <div>Hello world - fourth method - Functional Component</div>
);

export default WelcomFunction;
