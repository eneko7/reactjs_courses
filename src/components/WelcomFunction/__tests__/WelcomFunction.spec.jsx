import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import WelcomFunction from '../WelcomFunction';

const renderer = new ShallowRenderer();
renderer.render(<WelcomFunction />);
describe('WelcomFunction', () => {
  it('renders correctly', () => {
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
