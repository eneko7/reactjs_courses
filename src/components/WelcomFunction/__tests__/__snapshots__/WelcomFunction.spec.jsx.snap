// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomFunction renders correctly 1`] = `
<div>
  Hello world - fourth method - Functional Component
</div>
`;
